HISTORY.rst

```diff
@@ -2,6 +2,40 @@
 History
 =======
 
+3.26.0 (2021-01-08)
+-------------------
+
+Critical Changes:
+
+- CumulusCI can now resolve dependencies using second-generation packages (2GPs) for upstream projects. When a 2GP build runs on a release branch (starting with ``prefix/NNN``, where ``prefix`` is the feature branch prefix and ``NNN`` is an integer), CumulusCI will look for a matching release branch in each upstream dependency and use a 2GP package build on that release branch if present, falling back to a 1GP beta release if not present.
+
+Changes:
+
+- CumulusCI now reports how long it took for flows to run.
+
+- Flows ``ci_feature`` and ``ci_feature_beta_deps`` now only run the ``github_automerge_feature`` task if the branch begins with the configured feature branch prefix.
+
+- Running the ``deploy`` task with the ``path`` option set to a path that doesn't exist will log a warning instead of raising an error.
+
+- CumulusCI now installs beta dependencies when building 2GP feature test and QA orgs.
```

Comment:
This might be clearer: "When the ci_feature_2gp and qa_org_2gp flows install dependencies, the latest beta version will be used when available."